### ISSUE TITLE
Menu configuration > services : losing filters

### ISSUE THREAD

**Issue by pkriko** [centreon/centreon#7734]
# BUG REPORT INFORMATION

### Prerequisites

> The opened issue, must be code related. GitHub is not meant for support. Feel free to check the CONTRIBUTING section for more details.

***Versions***

For the RPM based systems

-- Copy/Paste the result of the following command --
```
centreon-clib-19.04.0-1.el7.centos.x86_64
centreon-connector-perl-19.04.0-2.el7.centos.x86_64
centreon-widget-servicegroup-monitoring-19.04.0-7.el7.centos.noarch
centreon-broker-storage-19.04.0-2.el7.centos.x86_64
centreon-engine-extcommands-19.04.1-9.el7.centos.x86_64
centreon-awie-19.04.0-1.el7.centos.noarch
centreon-trap-19.04.2-7.el7.centos.noarch
centreon-widget-grid-map-19.04.0-5.el7.centos.noarch
centreon-base-config-centreon-engine-19.04.2-7.el7.centos.noarch
centreon-bam-server-19.04.0-3.el7.centos.noarch
centreon-license-manager-19.04.0-1.el7.centos.noarch
centreon-mbi-release-19.04-1.el7.centos.noarch
centreon-license-manager-common-19.04.0-1.el7.centos.noarch
centreon-broker-19.04.0-2.el7.centos.x86_64
centreon-widget-engine-status-19.04.0-5.el7.centos.noarch
centreon-widget-host-monitoring-19.04.0-5.el7.centos.noarch
centreon-fingerprint-19.04.0-1.el7.centos.x86_64
centreon-auto-discovery-server-19.04.0-4.el7.centos.x86_64
centreon-common-19.04.2-7.el7.centos.noarch
centreon-database-19.04.2-7.el7.centos.noarch
centreon-bi-server-19.04.0-1.el7.centos.noarch
centreon-widget-live-top10-memory-usage-19.04.0-5.el7.centos.noarch
centreon-bam-release-19.04-1.el7.centos.noarch
centreon-web-19.04.2-7.el7.centos.noarch
centreon-connector-19.04.0-2.el7.centos.x86_64
centreon-broker-core-19.04.0-2.el7.centos.x86_64
centreon-perl-libs-19.04.2-7.el7.centos.noarch
centreon-engine-19.04.1-9.el7.centos.x86_64
centreon-broker-cbd-19.04.0-2.el7.centos.x86_64
centreon-widget-hostgroup-monitoring-19.04.0-5.el7.centos.noarch
centreon-map-release-19.04-1.el7.centos.noarch
centreon-release-19.04-1.el7.centos.noarch
centreon-widget-graph-monitoring-19.04.0-5.el7.centos.noarch
centreon-pp-manager-19.04.0-3.el7.centos.noarch
centreon-map-web-client-19.04.2-1.el7.centos.noarch
centreon-poller-centreon-engine-19.04.2-7.el7.centos.noarch
centreon-connector-ssh-19.04.0-2.el7.centos.x86_64
centreon-widget-service-monitoring-19.04.0-5.el7.centos.noarch
centreon-broker-cbmod-19.04.0-2.el7.centos.x86_64
centreon-widget-tactical-overview-19.04.0-7.el7.centos.noarch
centreon-19.04.2-7.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-19.04.0-5.el7.centos.noarch
centreon-engine-daemon-19.04.1-9.el7.centos.x86_64

```

***Operating System***

*CentOS, Debian ...*

CentOS 7.6

***Browser used***

- [ x] Google Chrome
- [ x] Firefox
- [ ] Internet Explorer IE11
- [ ] Safari

Version: --
Chrome 75.0.3770.100
Firefox 60.7.2esr

***Additional environment details (AWS, VirtualBox, physical, etc.):***

### Description

-- Describe the encountered issue --

When I filter my service by using a hostname and I want to go to page 2, I lost my filter ...

### Steps to Reproduce

step : 1
![image](https://user-images.githubusercontent.com/32265250/61933348-f8ba9780-af85-11e9-8216-8e10dd242ee9.png)
step 2 : I click on page 2 

### Describe the received result

I lose my filter : 
![image](https://user-images.githubusercontent.com/32265250/61933438-24d61880-af86-11e9-9beb-27325e381f05.png)

### Describe the expected result
To go on the page 2 with my filter

### Logs

Unfortunately nothing relevant in the php logs or the web console.

**Comment by adr-mo** [comment]
Hey it is currently being fixed :)

**Comment by pkriko** [comment]
Lourd Bro !

**Comment by lpinsivy** [comment]
solved in Centreon 19.10.0
